x-pack/plugins/security_solution/public/detections/components/rules/description_step/helpers.tsx

```diff
@@ -219,11 +224,75 @@ export const buildStringArrayDescription = (
   return [];
 };
 
-export const buildSeverityDescription = (label: string, value: string): ListItems[] => [
+const OverrideColumn = styled(EuiFlexItem)`
+  width: 125px;
+  max-width: 125px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+`;
+
+export const buildSeverityDescription = (severity: AboutStepSeverity): ListItems[] => [
   {
-    title: label,
```

Comment:
Does this fix that UI warning about title being undefined?

Reply:
Yeah, I think that should take care of it. Was a little split on pulling in the `i18n` from the fields vs leveraging the forms schema label. I think it'll be best to use the label from the schema, but will need to re-work the data model for these complex fields once the hooksform fixes land.